backend/src/requestTypes.ts

```diff
@@ -19,6 +21,27 @@ export interface LoginBody {
   userType: UserType;
 }
 
+export interface UserIdBody {
+  userId: number
+}
+
+export interface CreateEventBody {
+  banner: string;
+  name: string;
+  startDateTime: Dayjs; 
```

Comment:
i'd recommend instead using the native Date type. you can use the `.toDate()` method on a `dayjs` object to convert it to this